torch/_dynamo/config.py

```diff
@@ -756,6 +756,10 @@ def default_debug_dir_root() -> str:
 # Deprecated! Please use the config in torch/fx/experimental/_config instead.
 enrich_profiler_metadata: bool = False
 
+# Experimental flag to enable regional compile on invoke_subgraph HOP.
+# For testing only!
+enable_invoke_subgraph_regional_compile: bool = False
```

Comment:
What does it mean to "enable regional compile on invoke_subgraph HOP"? We error out if someone provides aot_config? or something else?

Reply:
yes. Because this feature is still in implementation, we do not want people to use it yet. So if anyone provide a config/option to nested_regional_compile, we error out, unless they set this config.